apps/react-nextjs/pages/index.tsx

```diff
@@ -24,6 +24,8 @@ function Home() {
           <Link href="/mfa">{`<MultiFactorAuth />`}</Link>
           <br />
           <Link href="/step-up-auth">{`<StepUpAuth />`}</Link>
+          <br />
+          <Link href="/gdpr-consent-dialog">{`<GDPRConsentDialog />`}</Link>
```

Comment:
The idea was to import the route string from the relevant page module - looks like we haven't done this yet, but maybe we should start here by importing the route from `GDPRConsentDialogPageRoute`. cc @kaspermroz